core/pkg/ingress/controller/controller.go

```diff
@@ -671,29 +671,6 @@ func (ic *GenericController) getBackendServers(ingresses []*extensions.Ingress)
 	return aUpstreams, aServers
 }
 
-// GetAuthCertificate ...
-func (ic GenericController) GetAuthCertificate(secretName string) (*resolver.AuthSSLCert, error) {
```

Comment:
You cannot remove this function. It's used in the auth cert annotation parser

Reply:
I see and it is in ingress-controller because it deals with secret objects part of the ingress controller.